client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx

```diff
@@ -112,35 +114,24 @@ export const ApplicationDetailDrawer: React.FC<
                 </>
               ) : null}
             </List>
-            <Title headingLevel="h3" size="md">
-              {t("terms.businessService")}
-            </Title>
-            <Text component="small">
-              {application?.businessService ? (
-                <ApplicationBusinessService
-                  id={application.businessService.id}
-                />
-              ) : (
-                t("terms.unassigned")
-              )}
-            </Text>
-            <Title headingLevel="h3" size="md">
-              {t("terms.migrationWave")}
-            </Title>
-            <Text component="small">
-              {application?.migrationWave
-                ? application.migrationWave.name
-                : t("terms.unassigned")}
-            </Text>
             <Title headingLevel="h3" size="md">
               {t("terms.effort")}
             </Title>
             <Text component="small">
-              {application?.effort ?? t("terms.unassigned")}
+              <Text component="small">
+                {application?.effort !== 0
+                  ? application?.effort
+                  : t("terms.unassigned")}
```

Comment:
If `effort` is undefined (i.e. omitted from hub's response), undefined !== 0 and then will render undefined and just show up as blank.